nilearn/datasets.py

```diff
@@ -792,10 +792,10 @@ def fetch_yeo_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     data: sklearn.datasets.base.Bunch
         dictionary-like object, keys are:
 
-        - "tight_7", "liberal_7": 7-region parcellations, resp. tightly
-          fitted to cortex shape, and liberally fitted.
+        - "thin_7", "thick_7": 7-region parcellations,
```

Comment:
I had a  quick look at the paper, and did not find these names. However, they are present in the following page
http://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation_Yeo2011
I do not have any strong opinion: the names are probably slightly better. Do we need to have some kind of compatibility with the freesurfer site ?